python/lib/evaluations/dmod/evaluations/backends/file.py

```diff
@@ -128,6 +149,7 @@ def write_to_hdf(
             generated_file.attrs['standard_deviation'] = data.standard_deviation
             generated_file.attrs['location_count'] = len(data)
 
+    @deprecated
```

Comment:
Maybe? That tag is mostly there because I'd like to move the logic elsewhere because it doesn't belong there, but I don't have the time to move everything safely.